modules/web/src/assets/config/changelog.json

```diff
@@ -1,9 +1,17 @@
 {
-  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.25.md#v2253",
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.25.md#v2254",
   "entries": [
     {
       "category": "fixed",
-      "description": "[ACTION REQUIRED] The latest Ubuntu 22.04 images ship with cloud-init 24.x package. This package has breaking changes and thus rendered our OSPs as incompatible. It's recommended to refresh your machines with latest provided OSPs to ensure that a system-wide package update, that updates cloud-init to 24.x, doesn't break the machines."
+      "description": "Fix a bug where unrequired  `cloud-config` secret was being propagated to the user clusters"
```

Comment:
I don't think code blocks work here, so please remove the back ticks and check the spaces before that as well.

Reply:
Plus, this should have a period in the end (like the other two entries).